test/TesterInternal/StreamingTests/StreamLimitTests.cs

```diff
@@ -55,21 +53,20 @@ public override TestCluster CreateTestCluster()
             return new TestCluster(options);
         }
 
-
         public StreamLimitTests(ITestOutputHelper output)
         {
             this.output = output;
             StreamNamespace = StreamTestsConstants.StreamLifecycleTestsNamespace;
             mgmtGrain = GrainClient.GrainFactory.GetGrain<IManagementGrain>(0);
         }
         
-        [Fact, TestCategory("Streaming"), TestCategory("Limits")]
```

Comment:
same here why remove streaming and limits tag? especially why remove streaming tag?